wai-middleware-prometheus/wai-middleware-prometheus.cabal

```diff
@@ -1,5 +1,5 @@
 name:                wai-middleware-prometheus
-version:             0.3.0
+version:             0.3.1
```

Comment:
There are a bunch of API breaking changes incoming to this project at the moment. If this isn't urgent, we can bundle this with the changes from #25 into a single version bump.

If that's OK, can you revert this and then add a line under the `CHANGELOG.md` file under the `HEAD` line.